ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp

```diff
@@ -23,13 +23,15 @@ struct OutstandingMessage
 static LIST_HEAD(OutstandingMessage_Head, OutstandingMessage) s_outstandingMessages = LIST_HEAD_INITIALIZER(OutstandingMessage_Head);
 static Mutex s_outstandingMessagesMutex = {};
 static atomic_uint_least64_t s_nextMessageId;
+static atomic_uint_least64_t s_messagesSent;
 static volatile bool s_isShuttingDown;
 
 
 bool ProviderMessaging_Init()
 {
     LIST_INIT(&s_outstandingMessages);
-    s_nextMessageId = 1;
+    s_nextMessageId = 0;
```

Comment:
Why was `s_nextMessageId` changed from 1 to 0?

Reply:
It could have stayed at 1.